Source/WebKit/Platform/IPC/IPCSemaphore.h

```diff
@@ -76,8 +73,11 @@ class Semaphore {
     explicit operator bool() const { return m_sendRight || m_semaphore != SEMAPHORE_NULL; }
 #elif OS(WINDOWS)
     explicit Semaphore(Win32Handle&&);
+    Win32Handle toWin32Handle();
```

Comment:
I'm not sure "to" is a good description of what this accessor is doing.  Also, this could probably be inline in the header.

Reply:
Ok, I'm still fighting to get this particular getter to build with the wincairo EWS. I'll refine.